src/compiler/diagnosticMessages.json

```diff
@@ -2525,6 +2525,10 @@
         "category": "Error",
         "code": 2741
     },
+    "The inferred type of '{0}' requires a reference to a module at '{1}' to name, which would likely break on type redistribution. A type annotation is necessary.": {

```

Comment:
```
Naming the type of '{0}' requires a reference to a module at '{1}' to name, which would likely break on type redistribution. A type annotation is necessary.
```

or

``
Generating a declaration for the type of '{0}' requires a reference to a module at '{1}' to name, which would likely break on type redistribution. A type annotation is necessary.
```

Reply:
`The inferred type of '{0}' cannot be named without a reference to '{1}'. This is likely not portable. A type annotation is necessary.` ?